fiftyone/utils/iou.py

```
@@ -1001,18 +1011,26 @@ def _compute_keypoint_similarities(preds, gts, classwise=False, sparse=False):
     return sims
 
 
-def _compute_object_keypoint_similarity(gtp, predp):
+def _compute_object_keypoint_similarity(gtp, predp, keypoint_sigmas=None):
     gtp = np.asarray(gtp, dtype=float)
     predp = np.asarray(predp, dtype=float)
 
+    if keypoint_sigmas is not None:
+        if len(gtp) != len(keypoint_sigmas):
```

Ruff (0.15.17)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)